payments-core/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -241,7 +243,10 @@ sealed interface StripeIntent : StripeModel {
             }
         }
 
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
         @Parcelize
-        internal data class WeChatPayRedirect(val weChat: WeChat) : NextActionData()
+        data class WeChatPayRedirect(val weChat: WeChat) : NextActionData()
     }
 }
+
+fun StripeIntent.getRequestCode() = StripePaymentController.getRequestCode(this)
```

Comment:
This isn't really related to the Intent object - I don't think it should go here.

Reply:
Without this extension, we'll have to expose `StripePaymentController` to extract the request code. Created a dedicated `StripeIntentKtx` file to host this method